src/Http/Place/ImportPlaceRequestHandler.php

```diff
@@ -224,13 +225,12 @@ public function handle(ServerRequestInterface $request): ResponseInterface
             $commands[] = new DeleteCurrentOrganizer($placeId);
         }
 
-        $lastCommandId = null;
         foreach ($commands as $command) {
-            /** @var string|null $commandId */
-            $commandId = $this->commandBus->dispatch($command);
-            if ($commandId) {
-                $lastCommandId = $commandId;
+            try {
+                $commandId = $this->commandBus->dispatch($command);
+            } catch (NotAllowedToPublish $notAllowedToPublish) {
             }
+            $lastCommandId = $commandId ?? null;
```

Comment:
Nice, I didn't realize you could also do this to check possibly undefined variables

Reply:
Some sugar since PHP 7.0: https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op

Reply:
But in this case healthy for the code 😀

Reply:
I knew about the `??` operator but not that you could also use it for undefined variables, I thought it only worked for arrays/objects like `$arr['key'] ?? : null`. But makes sense if it's basically a shorthand for `isset()`